app/helpers/application_helper/toolbar/middleware_server_center.rb

```diff
@@ -100,7 +100,7 @@ class ApplicationHelper::Toolbar::MiddlewareServerCenter < ApplicationHelper::To
           :data => {'toggle'        => 'modal',
                     'target'        => '#modal_d_div',
                     'function'      => 'miqCallAngular',
-                    'function-data' => '{"name": "showListener", "args": []}'})
+                    'function-data' => '{\"name\": \"showListener\", \"args\": []}'})
```

Comment:
Aah, so we need to properly escape it when putting it in the html, not here. `function-data` should be a proper json, without extra backslashes, there's probably a missing `j_str` where you putting it in the html..